### PR TITLE
Enable checker in captplanet and staging

### PR DESCRIPTION
**Pull request by jenlij**
Enable checker in captplanet and staging

### CODE REVIEW

File: pkg/datarepair/repairer/repairer.go [hunk 1]
```diff
@@ -17,6 +17,7 @@ import (
 
 // Repairer is the interface for the data repairer
 type Repairer interface {
+	//do we need this method? It doesn't look implemented
```
Comment (cam-a) [inline pkg/datarepair/repairer/repairer.go]: You're right. We don't need that. Good catch!

File: pkg/datarepair/checker/checker.go [hunk 2]
```diff
@@ -82,7 +82,16 @@ func (c *checker) identifyInjuredSegments(ctx context.Context) (err error) {
...
 					return Error.New("error unmarshalling pointer %s", err)
 				}
-				pieces := pointer.Remote.RemotePieces
+				remote := pointer.GetRemote()
+				if remote == nil {
+					c.logger.Debug("no remote segment on pointer")
+					continue
+				}
+				pieces := remote.GetRemotePieces()
+				if pieces == nil {
```
Comment (cam-a) [inline pkg/datarepair/checker/checker.go]: I wonder if it's okay to have a remote segment with no pieces... Do you think that should be an error?

Comment (dylanlott) [reply pkg/datarepair/checker/checker.go]: If there are no remote pieces, that just means it's an inline segment, so I think it's fine.

Comment (cam-a) [reply pkg/datarepair/checker/checker.go]: ok

**Pull request merged**